resources/views/user/show.blade.php

```diff
@@ -1,18 +1,54 @@
 @extends('layouts.master')
 
 @section('title', $user->name . '\'s userpage')
-
+    <style>
```

Comment:
this shit is put above the doctype declaration
![something like this](https://cdn.discordapp.com/attachments/420215421229334528/422807257424855040/unknown.png)
needs a new section in `/resources/views/layouts/master.blade.php` 
put an `@yield('custom-css')` between `<head>` and `</head>` and then wrap those in `@section('custom-css')` and `@endsection`